app/validation/validator.py

```diff
@@ -363,6 +364,16 @@ def _validate_questions(self, block_or_variant, numeric_answer_ranges):
                             self._validate_minimum_and_maximum_offset_date(answer)
                         )
 
+                if answer["type"] == "TextField":
+                    if "suggestions_url" in answer and not self._validate_url(
+                        answer["suggestions_url"]
+                    ):
+                        errors.append(
+                            self._error_message(
+                                f'Answer lookup url used for TextField `{answer["id"]}` is invalid'
```

Comment:
We should be consistent in how we refer to the suggestions url, this should read "Suggestions url used for ..."